docs/how_tos/MonoDepth_how_to.md

```diff
@@ -61,7 +61,7 @@ Install required Python modules, for example, kiwisolver or cycler, if you get m
 
 Use your input image:
 ```
-python3 $OV/inference_engine/demos/python_demos/monodepth_demo/monodepth_demo.py -m $WD/public/midasnet/FP32/midasnet.xml -i input-image.jpg
+python3 $OV/inference_engine/demos/python_demos/monodepth_demo/monodepth_demo -m $WD/public/midasnet/FP32/midasnet.xml -i input-image.jpg
```

Comment:
please do not change it. I believe it should be done separately in the scope of other task